app/src/main/java/org/simple/clinic/registration/confirmfacility/ConfirmFacilitySheet.kt

```diff
@@ -29,8 +31,33 @@ class ConfirmFacilitySheet : BottomSheetActivity() {
     }
   }
 
+  private val facilityName: String by lazy {
+    intent.getStringExtra(EXTRA_FACILITY_NAME)
+  }
+
+  private val facilityUuid: UUID by lazy {
+    intent.getSerializableExtra(EXTRA_FACILITY_UUID) as UUID
+  }
```

Comment:
We can use `by unsafeLazy` here since we will only invoke this from the UI thread.